app/services/token_requester.rb

```diff
@@ -0,0 +1,21 @@
+class TokenRequester
+  def initialize(request_headers)
+    @request_headers = request_headers
+  end
+
+  def auth_token
+    token = get_token_from_headers
+
+    return unless token
```

Comment:
https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition

`return unless (token = get_token_from_headers)`